README.md

```diff
@@ -1,7 +1,6 @@
 # Spine Generic Public Database (Single Subject)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4299148.svg)](https://doi.org/10.5281/zenodo.4299148)
-[![Build Status](https://travis-ci.com/spine-generic/data-single-subject.svg?branch=master)](https://travis-ci.com/spine-generic/data-single-subject)
-
+![Python application](https://github.com/spine-generic/data-single-subject/workflows/Python%20application/badge.svg)
```

Comment:
the URL might need to be adapted with the suggested name "BIDS Validator"

Reply:
done